notification-service/README.md

```diff
@@ -0,0 +1,77 @@
+# Unified Auctions Notification Service
+
+## What is "Unified Auctions Notification Service?"?
+
+"Unified Auctions Notification Service" is a server-side application with the purpose of updating users to a specific
+blockchain contract. Whenever a change is made to this contract the user will be informed via email or other methods.
+This is to ensure quick response times, when changes were made that could break our products.
+
+The bots can be hosted by any user, provided, they configure related environment variables. For more details, please
+refer to [`Environment variables`](#environment-variables) section below.
+
+## Setup
+
+```bash
+# install dependencies
+$ npm install
+
+# serve with hot reload
+$ npm run dev
+
+# build for production and launch server
+$ npm run build
+$ npm run start
+```
+
+## Environment variables
+
+- `INFURA_PROJECT_ID`: (required) [infura](https://infura.io/) project id (can be found in: dashboard -> ethereum ->
+- `ETHERSCAN_API_KEY`: (required) [Etherscan API Key](https://docs.etherscan.io/getting-started/viewing-api-usage-statistics), which is used to automatically download ABIs 
+- `ETHEREUM_NETWORK`: (optional, default `kovan`) – internal network name on which the bot poll for auctions. Available
+- `SMTP_*`: (required) - SMTP data for the service to send emails
+    - `SMTP_HOST`: (required) - SMTP host address
+    - `SMTP_PORT`: (optional) - SMTP port. Defaults to `465`
+    - `SMTP_USERNAME`: (required) - SMTP username
+    - `SMTP_PASSWORD`: (required) - SMTP password
+    - `SMTP_EMAIL`: (required) - The outgoing address which is displayed to the user
+- `RECEIVERS`: (required) - a json-formatted object with `email` as key and `Event Subscription Ids` as an array value, eg `RECEIVERS="{\"email@example.com\": [\"EVENT_1\", \"EVENT_2\"]}"`. The email will then become subscribed to the listed events
+
+Note: env variables are accessible via the `secret` command under `auction-ui/notification-service/${environment}`.
```

Comment:
The path to the secrets should be `auction-ui/${host}/notification-service`. 

But I couldn't find any secrets neither under `auction-ui/notification-service/localhost` or under `auction-ui/notification-service/dev`. Where are they currently stored?

Reply:
I have not added any secrets yet, as we need to generate new API tokens for etherscan. Who should take care of this?

Reply:
I took that one over and created an API key for us. You can find credentials in bitwarden under "SAS Etherscan Account"

Reply:
Done! ✅ Also changed in the bot readme